public/index.html

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
+<html lang="en-us">
+<head>
+  <title>avro-schema-registry</title>
+  <style type="text/css">
+    body {
+      background-color: #444;
+      color: #FFF;
+      font-family: "proxima-nova", helvetica, verdana, arial, sans-serif;
+      margin: 4%;
+      padding: 16px;
+      font-size: 1.75em;
+      text-rendering: optimizeLegibility;
+      font-feature-settings: "kern" 1;
+      font-kerning: normal;
+    }
+    h1 {
+      margin: 1.5em 0 0;
+    }
+    p {
+      font-weight: lighter;
+      line-height: 120%;
+    }
+    a {
+      color: inherit;
+    }
+    #container {
+      padding:10px;
+    }
+    .content {
+      padding-top:0em;
+    }
+  </style>
+</head>
+<body>
+  <h1>avro-schema-registry</h1>
+  <p>
+    <strong>
+      A service for storing and retrieving versions of Avro schemas
+    </strong>
+  </p>
+
+  <h2>Description</h2>
+  <p>
+    This service implements the API of the <a href="http://docs.confluent.io/3.0.0/schema-registry/docs/api.html" target="_blank">Confluent Schema Registry</a>.
+    <br/>
+    Imitation is the sincerest form of flattery, right?
+  </p>
+
+  <p>This service is hosted by <a href="http://www.salsify.com">Salsify</a>.</p>
```

Comment:
Should we include any kind of text on how you host/manage your own version of this service?